src/Propel/Generator/Manager/ReverseManager.php

```diff
@@ -202,7 +202,7 @@ protected function buildModel()
 
 
         $database = new Database($this->getDatabaseName());
-        $database->setPlatform($config->getConfiguredPlatform($connection), $databaseName);
+        $database->setPlatform($config->getConfiguredPlatform($connection));
```

Comment:
I was worried about this because it highlights something that looks like a hidden bug. It seems related to 268f44d9 .

But even if it is a bug we can't fix it in this PR.